docker/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3"
+
+services:
+  clio:
+    image: ${CLIO_IMAGE}
```

Comment:
Why is it better to have `docker-compose.yml` + `.env` instead of single `docker-compose.yml`?

Reply:
I don't think it adds anything really.
Definitely was very close to removing that so I will.